docs/release-notes.rst

```diff
@@ -6,6 +6,97 @@
+      ``det user change-password`` commands.
+   -  Existing users with empty or non-compliant passwords can still sign in. However, we recommend
+      updating these passwords to meet the new requirements as soon as possible.
+
+-  CLI: When creating non-remote users with ``det user create``, setting a password is now
+   mandatory.
+
+   -  You can set the password interactively by following the prompts during user creation.
+   -  Alternatively, set the password non-interactively using the ``--password`` option.
+   -  This requirement does not apply to users created with the ``--remote`` option, as Single
+      Sign-On will be used for these users.
+
+**New Features**
+
+-  Kubernetes: In the enterprise edition, add the ability to set up the Determined master service on
+   one Kubernetes cluster and manage workloads across different Kubernetes clusters. Additional
+   non-default resource managers and resource pools are configured under the master configuration
+   options ``additional_resource_managers`` and ``resource_pools`` (additional resource managers are
+   required to have at least one resource pool defined). Additional resource managers and their
+   resource pools must have unique names. For more information, visit :ref:`master configuration
+   <master-config-reference>`. Support for notebooks and other workloads that require proxying is
+   under development.
+
+-  WebUI: In the enterprise edition, add the ability to view resource manager name for resource
+   pools.
+
+-  API/CLI/WebUI: In the enterprise edition, route any requests to resource pools not defined in the
+   master configuration to the default resource manager, not any additional resource manager, if
+   defined.
+
+-  Configuration: In the enterprise edition, add a ``name`` and ``metadata`` field to the resource
+   manager section in the master configuration. Add an ``additional_resource_managers`` section that
+   follows the ``resource_manager`` and ``resource_pool`` configuration pattern.
+
+**Improvements**
+
+-  Configuration: The master configuration parameter ``observability.enable_prometheus`` now
+   defaults to ``true``. Consequently, Prometheus endpoints are enabled by default, which does not
+   affect clusters that do not use Prometheus.
+
+**Bug Fixes**
+
+-  Since version 0.27.0, the selection functionality in the Experiments page had a bug where it
+   attempts to perform actions on a larger set of experiments than selected on the screen. For
+   instance, if you try to move all experiments in one project into another using Select All ->
+   Actions -> Move, it may inadvertently include all experiments from the project also from other
+   projects you have access to edit. We urge all users on the affected versions to upgrade as soon
+   as possible.
+
+   -  There is a risk of data loss if attempting to delete a set of experiments delete and it
+      inadvertently deletes a larger set than intended.
+
```

Comment:
- There is a risk of data loss if, when attempting to delete a set of experiments, the action inadvertently deletes a larger set than intended.